django/db/backends/postgresql/base.py

```diff
@@ -212,7 +260,17 @@ def get_connection_params(self):
 
     @async_unsafe
     def get_new_connection(self, conn_params):
-        connection = Database.connect(**conn_params)
+        if self.is_psycopg3:
+            ctx = get_adapters_template(settings.USE_TZ, self.timezone)
+            connection = Database.connect(**conn_params, context=ctx)
+        else:
+            connection = Database.connect(**conn_params)
```

Comment:
```suggestion
            connection = self.Database.connect(**conn_params, context=ctx)
        else:
            connection = self.Database.connect(**conn_params)
```

I think, better to use `self.Dataabase` cause someone may extend this backend and override the `Database`